### ISSUE TITLE
Can't import file in online editor

### ISSUE THREAD

**Issue by JuliCardenasG** [swagger-api/swagger-editor#1716]
When I try to import a file to the Swagger online editor, it does nothing and shows a console error. I tried in both Chrome and Firefox in their latest versions using Ubuntu but It did not work in any of them.

| Q                                   | A
| ----------------------------------- | -------
| Bug or feature request?             |  Bug
| Which Swagger/OpenAPI version?      | 3.0.1
| Which Swagger-Editor version?       | Latest
| How did you install Swagger-Editor? | Not installed. Online editor.
| Which browser & version?            | Chrome 65 / Firefox 59
| Which operating system?             | Ubuntu 16.04

### Demonstration API definition

Error shown in Chrome console:
```
Uncaught TypeError: Cannot read property 'show' of undefined
    at r.showModal (topbar.jsx:251)
    at Object.n (ReactErrorUtils.js:24)
    at s (EventPluginUtils.js:83)
    at Object.a [as executeDispatchesInOrder] (EventPluginUtils.js:106)
    at p (EventPluginHub.js:41)
    at g (EventPluginHub.js:52)
    at Array.forEach (<anonymous>)
    at r (forEachAccumulated.js:22)
    at Object.processEventQueue (EventPluginHub.js:252)
    at n (ReactEventEmitterMixin.js:15)
```

In Firefox console:
```
TypeError: r.refs[(intermediate value)] is undefined[Saber más]
topbar.jsx:251:4
t/r.showModal
topbar.jsx:251:4
n
ReactErrorUtils.js:24
s
EventPluginUtils.js:83
a
EventPluginUtils.js:106
p
EventPluginHub.js:41
g
EventPluginHub.js:52
forEach self-hosted:271:13 r
forEachAccumulated.js:22
processEventQueue
EventPluginHub.js:252
n
ReactEventEmitterMixin.js:15
handleTopLevel
ReactEventEmitterMixin.js:25
o
ReactEventListener.js:70
perform
Transaction.js:141
batchedUpdates
ReactDefaultBatchingStrategy.js:60
o
ReactUpdates.js:95
dispatchEvent
ReactEventListener.js:145
dispatchEvent
```

### Expected Behavior
It should show a modal or ask for an input file to import.

### Current Behavior
It does nothing and shows a console error.

### Possible Solution
Maybe a React template error?

### Context
I edited an OpenAPI yaml file a couple days ago with the online editor and saved it to my computer. Today I tried to resume my work but the online editor doesn't let me import files.

**Comment by JuliCardenasG** [comment]
You're right @hkosova. Looks like the same issue. Waiting for the online editor update. Thanks.

**Comment by shockey** [comment]
Sorry for the delay - there were some issues with the Docker image builder, which was holding up the Swagger.io update.

Everything's resolved now, 3.5.1 is live at editor.swagger.io :smile:
